src/Core/src/ImageSources/StreamImageSourceService/StreamImageSourceService.iOS.cs

```diff
@@ -30,7 +30,7 @@ public partial class StreamImageSourceService
 				if (data == null)
 					throw new InvalidOperationException("Unable to load image stream data.");
 
-				var image = UIImage.LoadFromData(data, scale);
+				var image = UIImage.LoadFromData(data);
```

Comment:
This was always incorrect, but we never noticed because we always loaded with a 1.0 scale. According to the Apple docs:

> **scale**
> The scale factor to use when interpreting the image data. Specifying a scale factor of 1.0 results in an image whose size matches the pixel-based dimensions of the image. Applying a different scale factor changes the size of the image as reported by the [size](https://developer.apple.com/documentation/uikit/uiimage/1624105-size) property.

This means that an image file of 100x100 that is saved to disk will be loaded as 50x50 with a scale of 2. As a result, the UI then renders the image smaller than expected because we loaded the "actual size" as 50x50.

The scale parameter in this method is the scale of the "displaying" context - such as the screen. Not the scale of the image. This is why when we render fonts we use this parameter. We are asked to generate a font image for a display that has a scale of X.

Reply:
Can we add a comment to the code?